src/main/target/system_stm32h7xx.c

```diff
@@ -0,0 +1,981 @@
+/**
+  ******************************************************************************
```

Comment:
Not sure we want to stick these into `src/main/target` - we already made an effort to move the linker scripts out of there. How about `src/main/hal` or similar?

Also, note that this is adding two `system_stm32h7xx.c` files. Maybe rename the one in 	`src/main/drivers`?

Reply:
A relocation PR is ready to relocate linker `.ld` and `target/system_stm32h7xx.c`  which will be applied at the conclusion of the H7 code merge process. Relocating them at this point causes extra work on rebasing upcoming `EXST` PRs.

The "Two `system_stm32h7xx.c`" problem is common to all platforms. I think this should be handled separately when the H7 process is complete. I will add an issue ticket for reminder.